cql3/query_processor.cc

```diff
@@ -47,6 +47,7 @@ class query_backend::impl {
     wasmtime::Engine* _wasm_engine;
     wasm::instance_cache* _wasm_ic;
     wasm::alien_thread_runner* _wasm_atr;
+    locator::token_metadata_ptr _token_metadata;
```

Comment:
Shouldn't this be `const locator::shared_token_metadata&`? Token metadata is changing.